Bugsnag/Payload/BugsnagApp.m

```diff
@@ -32,7 +32,7 @@ + (BugsnagApp *)deserializeFromJson:(NSDictionary *)json {
         app.id = json[@"id"];
         app.releaseStage = json[@"releaseStage"];
         app.type = json[@"type"];
-        app.version = json[@"version"];
+        app.version = [json[@"version"] isKindOfClass:[NSString class]] ? json[@"version"] : nil;
```

Comment:
Why is this needed?

Reply:
It isn't - was left over from an earlier revision... corrected now 👍